profiles/sarika.json

```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://github.com/humsarika",
```

Comment:
kindly add `.png` infront of the link to get your profile image
```diff
- https://github.com/humsarika
+ https://github.com/humsarika.png
```

Reply:
I was trying to add a hosted image like others but couldn't able to do so. If I add  .png that will open my profile image not screenshot of my GitHub profile. So can I add this ---> "https://ibb.co/zFR8Hcb" link to image URL

Reply:
Hey @kaiwalyakoparkar here we need screenshot URL not the profile picture.